src/main/java/de/qabel/desktop/DesktopClientGui.java

```diff
@@ -55,63 +54,67 @@ public void start(Stage stage) throws Exception {
         runtimeConfiguration.setPrimaryStage(primaryStage);
         config = services.getClientConfiguration();
 
-        SceneAntialiasing aa = SceneAntialiasing.BALANCED;
-        primaryStage.getIcons().setAll(new Image(getClass().getResourceAsStream("/logo-invert_small.png")));
-        Scene scene;
-
-        Platform.setImplicitExit(false);
-        primaryStage.setTitle(getResources().getString("title"));
-        scene = new Scene(new LoginView().getView(), 370, 570, true, aa);
-        primaryStage.setScene(scene);
-
+        doLoginStage();
```

Comment:
do is superfluous. doCreateLoginStage? doShowLoginStage? you can just omit the `do` then